ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.9.yaml

```diff
@@ -0,0 +1,759 @@
+      - quay.io/edge-infrastructure/swagger-codegen-cli:2.4.18
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: assisted-service-build-base
+- dockerfile_literal: |
+    FROM base-build
+    COPY . .
+  inputs:
+    assisted-service-build-base:
+      as:
+      - base-build
+    src:
+      paths:
+      - destination_dir: ./
+        source_path: /go/src/github.com/openshift/assisted-service/.
+  to: assisted-service-build
+- dockerfile_path: Dockerfile.assisted-service
+  inputs:
+    openshift_release_golang-1.18:
+      as:
+      - registry.ci.openshift.org/openshift/release:golang-1.18
+  to: assisted-service
+- dockerfile_literal: |
+    FROM coreos/centos
+    COPY . .
+  from: src
+  to: assisted-service-scripts
+- dockerfile_literal: |
+    FROM coreos/centos
+    RUN yum install -y skopeo
+  from: src
+  to: assisted-service-publish-index
+operator:
+  bundles:
+  - dockerfile_path: deploy/olm-catalog/bundle.Dockerfile
+  substitutions:
+  - pullspec: quay.io/edge-infrastructure/assisted-service:ocm-2.9
+    with: assisted-service
+  - pullspec: quay.io/edge-infrastructure/assisted-installer:ocm-2.9
+    with: assisted-installer
+  - pullspec: quay.io/edge-infrastructure/assisted-installer-agent:ocm-2.9
+    with: assisted-installer-agent
+  - pullspec: quay.io/edge-infrastructure/assisted-installer-controller:ocm-2.9
+    with: assisted-installer-controller
+  - pullspec: quay.io/edge-infrastructure/assisted-image-service:ocm-2.9
+    with: assisted-image-service
+  - pullspec: quay.io/centos7/postgresql-12-centos7:latest
+    with: postgresql-12-centos7
+promotion:
+  name: ocm-2.9
+  namespace: edge-infrastructure
+releases:
+  candidate-4-11:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.11"
+  candidate-4-11-multi:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.11"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.14"
+  latest-multi:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.14"
+  stable-4-10:
+    integration:
+      name: "4.10"
+      namespace: ocp
+  stable-4-11:
+    integration:
+      name: "4.11"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: publish-python-client
+  postsubmit: true
+  steps:
+    test:
+    - ref: assisted-publish-python-client
+- as: mirror-nightly-image
+  cron: '@daily'
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: assisted-service
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.9
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: mirror-vcsref-image
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: assisted-service
+    env:
+      RELEASE_TAG_PREFIX: ocm-2.9
+    test:
+    - ref: assisted-baremetal-images-publish
+- as: push-pr-image
```

Comment:
not sure if we need it. I think it was mainly a hack back then to allow testing the publishing from a PR (when the code still needed more debugging)

Reply:
we still have it in 2.8/2.7